### DISCUSSION TITLE
阿里云open的OAuth验证API无法连接

### DISCUSSION THREAD

**Discussion by ghn9264** [AlistGo/alist#5306] [General]
9.28正常，9.29连不上了，是不是这两个接口都挂了？

**Comment by davidgodness** [comment]
无法获取刷新令牌？

**Reply by ghn9264** [reply]
![image](https://github.com/alist-org/alist/assets/32256957/e129d41e-8fcb-468f-9a2e-66a4bf979ea3)
![image](https://github.com/alist-org/alist/assets/32256957/7774baec-3dda-445d-8f90-01b7804b0115)

**Comment by ghn9264** [comment]
好像是阿里云盘网络波动？目前已恢复。

**Comment by xhofe** [comment]
原服务器被攻击。已切至阿里云

**Reply by ghn9264** [reply]
效率好高！谢谢
